addition of policy and alarm creation for asg

arnold1796 opened the pull request.

souradeepmoe reviewed (changes requested):
1.  Remove dynamic blocks for target tracking policies
2. Move `target` to tf-service moengage
3. Make all_alarms as map instead of list

arnold1796 commented:
> * Remove dynamic blocks for target tracking policies
> * Move `target` to tf-service moengage
> * Make all_alarms as map instead of list

1. Removed Dynamic block from target tracking policy.
2. "target" is moved to the main tf-service.
3. Removed addition of Custom Alarms, there is no use case to it as of now. Alarms is a map of objects.
4. variable autoscaling_policy enabled is required , when scenarios include that we need only ASG and not the policy/alarms. Keeping this in mind , the variable is indeed required.
5. Tested for ASG namespace and Queue namespace.
